api/src/org/labkey/api/util/ExceptionUtil.java

```diff
@@ -632,6 +632,20 @@ protected void renderView(Object model, HttpServletRequest request, HttpServletR
         }
     }
 
+    private static void logClientAbortException(Throwable ex)
+    {
+        String className = ex.getClass().getName();
+        String message = ex.getMessage();
+        Throwable cause = ex.getCause();
+
+        if (message != null) {
+            LOG.trace("Client abort exception: {} - {}", className, message);
+        } else if (cause != null) {
+            LOG.trace("Client abort exception: {} (cause: {})", className, cause.getClass().getName());
+        } else {
+            LOG.trace("Client abort exception: {}", className);
+        }

```

Comment:
Is there a reason to not just do the standard exception logging?

```suggestion
        LOG.trace("Client abort exception", ex);
```

Reply:
Ok I wasn't sure if we had to be more defensive since we're handling many type of exceptions here. Updated.